### PR TITLE
chore: remove metric from performance

### PR DESCRIPTION
**Pull request by ThibaultFy**
## Description

Remove metric from the performance data model

## Companion PR

- https://github.com/Substra/substra-backend/pull/650
- https://github.com/Substra/orchestrator/pull/213 (main, this PR)
- https://github.com/Substra/substra/pull/358

### CODE REVIEW

**Comment by Owlfred** [comment]
```
Usage: /e2e [options] [help]

/e2e may appear anywhere as long as it is on its own line

Options:
  --refs <value>                                         Extra refs (branch or tag) with format REPO=GIT_REF,REPO=GIT_REF.
  Supported repositories: hlf-k8s, orchestrator, substra-backend, substra-frontend, substra-tools, substrafl, substra, substra-tests, substra-ci.
  Example: /e2e --refs substra-backend=some_branch,orchestrator=some_tag (default: {})
  --tests-to-run, --tests <tests-to-run>                 Comma-separated list of tests to run. Valid options: sdk,substrafl,frontend or NONE. (default: "sdk")
  --benchmarks-to-run, --benchmarks <benchmarks-to-run>  Comma-separated list of workflows tests to run. Valid options: mnist, camelyon or NONE. (default: "NONE")
  --orchestrator-mode, --mode <orchestrator-mode>        Comma-separated list of orchestrator modes to run tests for. Valid options: standalone,distributed (default: "standalone,distributed")
  -h, --help                                             display help for command
```

**Comment by ThibaultFy** [comment]
/e2e --refs substra=remove-metric-from-performance,orchestrator=remove-metric-from-performance,substra-backend=remove-metric-from-performance

**Comment by ThibaultFy** [comment]
/e2e --refs substra=remove-metric-from-performance,orchestrator=remove-metric-from-performance,substra-backend=remove-metric-from-performance

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

File: server/standalone/migration/000057_remove-metric-from-performance.up.sql [hunk 1]
```diff
@@ -0,0 +1,23 @@
+SELECT execute($$
+        /* To keep differiantiability regarding the performances, we update the output identifier to the metric name */
+        /* on previous performances */
```
Comment (SdgJlbl) [inline server/standalone/migration/000057_remove-metric-from-performance.up.sql:3]: to remove?

**Pull request merged**